include/swift/AST/Module.h

```diff
@@ -1149,6 +1149,12 @@ class SourceFile final : public FileUnit {
     getInterfaceHash(str);
     out << str << '\n';
   }
+
+  /// Outputs Unified Symbol Resolutions (USRs) and a newline for each
+  /// function declaration in this source file that matches what XCTest
+  /// considers to be a viable test: a class instance method that takes no
+  /// parameters, returns void, and begins with "test".
+  void dumpXCTestMethods(llvm::raw_ostream &out);
```

Comment:
Makes sense to me! Just to explain my reasoning: I was copying the style of the `dump-` method immediately above this one, `SourceFile::dumpInterfaceHash(llvm::raw_ostream &)`. Still, I agree with your point. Maybe the above method should also be changed?